torch/csrc/lazy/ts_backend/view_ops/view.cpp

```diff
@@ -1,4 +1,4 @@
-#include <torch/csrc/lazy/core/view_ops/view.h>
+#include <torch/csrc/lazy/ts_backend/view_ops/view.h>
```

Comment:
Not a blocker, but @JackCaoG , have you decided how to handle view_ops for XLA? Leveraging @bdhirsh 's functionalization pass? Just wondering if view_ops should be shared or backend specific.